Messenger/Base.lproj/MainMenu.xib

```diff
@@ -368,8 +368,8 @@
                                     <action selector="performMiniaturize:" target="-1" id="VwT-WD-YPe"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="Zoom" id="R4o-n2-Eq4">
-                                <modifierMask key="keyEquivalentModifierMask"/>
+                            <menuItem title="Zoom" keyEquivalent="z" id="R4o-n2-Eq4">
+                                <modifierMask key="keyEquivalentModifierMask" control="YES" command="YES"/>
```

Comment:
Cmd+Ctrl+Z is the default shortcut for "Zoom," but this app doesn't have a full screen menu item (it should be under View), and the shortcut should be Cmd+Ctrl+F. I have another branch that adds a full-screen menu item as well as one for the main window if you close it for some reason. The shortcut for full-screen can be useful and I can submit a PR for that if you want.